SendEmailTool.py

```
@@ -1,36 +1,218 @@
+def _send_via_gmail_smtp(
+    gmail_user: str,
+    app_password: str,
+    to_email: str,
+    subject: str,
+    body: str,
+    app_name: str | None = None,
+    body_type: str = "plain",
+    text_body: str | None = None,
+) -> None:
+    if not gmail_user or not app_password:
+        raise ValueError("Gmail SMTP 邮件配置不完整，请检查 EMAIL.GMAIL_USER 和 EMAIL.APP_PASSWORD")
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)